scripts/fix_requirements.py

```
@@ -0,0 +1,48 @@
+    # Remove any self-references
+    lines = content.split('\n')
+    cleaned_lines = []
+    
+    for line in lines:
+        # Skip lines that reference themselves or create circles
+        if line.strip() in ['-r requirements.txt', '-r ./requirements.txt']:
+            cleaned_lines.append('# Removed circular self-reference')
+            continue
+        
+        # Skip duplicate -r requirements-locked.txt references
+        if line.strip() == '-r requirements-locked.txt' and any('-r requirements-locked.txt' in l for l in cleaned_lines):
```

Flake8 (7.2.0)

[error] ambiguous variable name 'l'

(E741)